packages/apps/graph/src/graph/Query/account.ts

```diff
@@ -0,0 +1,23 @@
+import { builder } from '../builder';
+import Account from '../objects/ModuleAccount';
+
+builder.queryField('account', (t) => {
+  return t.field({
+    args: {
+      accountName: t.arg.string({ required: true }),
+      moduleName: t.arg.string({ required: true }),
+    },
+    type: Account,
+    resolve: async (root, args) => {
```

Comment:
Maybe instead of `root` we could have `parent` for consistency purposes
```suggestion
    resolve: async (parent, args) => {
```

Reply:
Agreed 👍